core/codegen/module.go

```diff
@@ -58,6 +68,7 @@ func NewModule(name string, target *device.ABI) *Module {
 			Uint16:        Integer{false, basicType{"uint16", 16, ctx.Int16Type()}},
 			Uint32:        Integer{false, basicType{"uint32", 32, ctx.Int32Type()}},
 			Uint64:        Integer{false, basicType{"uint64", 64, ctx.Int64Type()}},
+			Uintptr:       Integer{false, basicType{uintptrName, ptrSize, ctx.IntType(ptrSize)}},
```

Comment:
If we name this "uintptr" it will not match with variables of the same type elsewhere.

Reply:
Specifically for extern functions, it will not match correctly.